cibuild.sh

```diff
@@ -299,12 +299,18 @@ if [[ $BOOTSTRAP_ONLY = true ]]; then
     exit $?
 fi
 
+# Microsoft.Net.Compilers package is available now, so we can use the latest csc.exe
+if [ "$host" = "Mono" ]; then
+        CSC_EXE=`find $PACKAGES_DIR/microsoft.net.compilers -name csc.exe`
+        CSC_ARGS="/p:CscToolExe=csc.exe /p:CscToolPath=`dirname $CSC_EXE` /p:DebugType=portable"
+fi
+
 # The set of warnings to suppress for now
 # warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.
 # warning MSB3026: Could not copy "XXX" to "XXX". Beginning retry 1 in 1000ms.
 # warning AL1053: The version '1.2.3.4-foo' specified for the 'product version' is not in the normal 'major.minor.build.revision' format
-_NOWARN="MSB3277;MSB3026;AL1053"
+_NOWARN="MSB3276;MSB3277;MSB3026;AL1053"
```

Comment:
FYI, it is being referenced [here](https://github.com/Microsoft/msbuild/blob/xplat/src/Utilities/project.json#L5).